src/Net/Http.fs

```diff
@@ -1513,6 +1513,12 @@ type Http private() =
 #endif
 
     static let charsetRegex = Regex("charset=([^;\s]*)", regexOptions)
+    
+    /// Correctly encodes large form data values.
+    /// See https://blogs.msdn.microsoft.com/yangxind/2006/11/08/dont-use-net-system-uri-unescapedatastring-in-url-decoding/
+    /// and https://msdn.microsoft.com/en-us/library/system.uri.escapedatastring(v=vs.110).aspx
+    static member internal UrlEncode (query:string) =
+        (WebUtility.UrlEncode query).Replace("+","%20")
```

Comment:
Is WebUtility.UrlEncode available on PCL 7 and PCL 259?

Reply:
[`WebUtility` is available in `Profile7`](https://stackoverflow.com/a/28974461). It's available when targeting .NET 4.5. What is FSharp.Data targeting? Also, do you use `System.Web.HttpUtility` anwhere? I assume not if you are targeting PCL.